mcpgrafana.go

```diff
@@ -135,6 +155,8 @@ var ExtractGrafanaClientFromEnv server.StdioContextFunc = func(ctx context.Conte
 		cfg.APIKey = apiKey
 	}
 
+	cfg.Debug = GrafanaDebugFromContext(ctx)
```

Comment:
should we pass all config options through env vars? wondering if using one way or the other will make it maintainable over time

Reply:
Yeah I do think we need to standardize here, and maybe revisit all of the config options to be honest. Ideally everything would be configurable via either env or CLI flag which we could get by using a more full featured CLI parsing library, but that feels like a future thing!

I chose a CLI flag because it feels similar to `--log-level` to me!